services/github/src/handlers.js

```diff
@@ -295,11 +295,17 @@ async function deprecatedStatusHandler(message) {
     do {
       let group = await this.queueClient.listTaskGroup(message.payload.taskGroupId, params);
       params.continuationToken = group.continuationToken;
-      group.tasks.forEach(task => {
-        if (['failed', 'exception'].includes(task.status.state)) {
+
+      for (let i = 0; i < group.tasks.length; i++) {
+        // don't post group status for checks API
+        if (group.tasks[i].task.routes.includes(this.context.cfg.app.checkTaskRoute)) {return;}
```

Comment:
I kinda thought eslint would complain, but IMHO the return statement should be on its own line

Reply:
Eslint complained only about my very first variant:
`if (blablabla) return;` <---- it wanted the return statement to be in curlies